### DISCUSSION TITLE
Nextflow + Nextflow Tower behind Proxy

### DISCUSSION THREAD

**Discussion by StephanDrukewitz** [nextflow-io/nextflow#3753] [Q&A]
Hello, 
Is it possible to use nextflow in combination with tower behind a proxy firewall ?  At the moment I always get

`Unexpected response for request https://api.tower.nf/`

As far I understand this message originates from nextflow.util.SimpleHttpClient, is there an option to provide cert files to nextflow or the tower plugin ?

**Comment by pditommaso** [comment]
Use HTTP_PROXY and HTTPS_PROXY env variables (either lower or upper case) 

https://www.cyberciti.biz/faq/linux-unix-set-proxy-environment-variable/
